index.js

```diff
@@ -1,5 +1,69 @@
 'use strict';
 
+var superagent = require('superagent');
+
+global.D2LAccessTokenExpiresAt = 0;
+
+function now() {
+	return Date.now()/1000 | 0;
```

Comment:
Are we worried at all about relying on the user's system clock for this? Seems... like it could cause a lot of strange bugs.